### PR TITLE
Add rust implementation

### PR DESCRIPTION
**Pull request by BigBlueHat**
Hoping to help promote more great community work. 😃

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
             <li>
               <a href="learn.html"><span class="icon-book"></span> Documentation</a>
             </li>
-            <!-- <li><a href="#"><span class="icon-beaker"></span> Developers</a></li> -->
+            <li><a href="#developers"><span class="icon-beaker"></span> Developers</a></li>
```
Comment (gkellogg) [inline index.html:61]: Does this need to be added to other files that can be navigated to?

Comment (gkellogg) [reply index.html:61]: Added developers link to other pages in https://github.com/json-ld/json-ld.org/pull/725/commits/a614b126dc813168b12957e259787c55b97c5493.

**Comment by BigBlueHat** [comment]
@gkellogg did you do a rebase for this "merge"? Seems like it, because all the sha's changed from my local copy's. No worries either way, just found it odd that my local checkout was now confused--thinking the branch wasn't merged...when clearly (from commit messages and contents) it had been.

**Comment by gkellogg** [comment]
I usually merge via rebar, but sometimes I miss it.

**Pull request merged**